docs/topics/developer-tools.rst

```diff
@@ -219,8 +238,7 @@ also request each page to get every quote on the site::
         def parse(self, response):
             data = json.loads(response.text)
             for quote in data["quotes"]:
-                quote = quote["text"]
-                print(quote)
+                yield {"quote": quote["text"]
```

Comment:
missing a curly brace: `yield {"quote": quote["text"]}`

Reply:
Whoops, sorry. Added it.